velox/exec/Spill.cpp

```diff
@@ -71,8 +68,9 @@ bool SpillFile::nextBatch(RowVectorPtr& rowVector) {
   if (input_->atEnd()) {
     return false;
   }
-  VectorStreamGroup::read(
-      input_.get(), &pool_, type_, &rowVector, &kDefaultSerdeOptions);
+  serializer::presto::PrestoVectorSerde::PrestoOptions options = {
```

Comment:
Can you help refactor VectorStreamGroup::read to take a const reference instead of a pointer? Thanks!

Reply:
No, because the input argument is `VectorSerde::Options*`, it maybe cast to subclass such as `PrestoOptions`. 
`struct PrestoOptions : VectorSerde::Options`